### PR TITLE
Add issue template for RFCs

### PR DESCRIPTION
**Pull request by Scrumplex**
To encourage more RFCs let's add an issue template.
This form uses the [template used by the Arch Linux project](https://gitlab.archlinux.org/archlinux/rfcs/-/blob/0ba3b61e987e197f8d1901709409b8564958f78a/rfcs/0000-template.rst).

---

How the form looks

### CODE REVIEW

**Comment by tonyaellie** [comment]
change Arch Linux to PolyMC on line 21?

File: .github/ISSUE_TEMPLATE/rfc.yml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  validations:
+    required: true
+- type: textarea
+  attributes:
+    label: Unresolved Questions
+    description: |
+      Are there any portions of your proposal which need to be discussed with the community before the RFC can proceed? 
+      Be careful here -- an RFC with a lot of remaining questions is likely to be stalled. 
+      If your RFC is mostly unresolved questions and not too much substance, it may not be ready.
+    placeholder: Does a substatial amount of users care about the cat?
```
Comment (ghost) [inline .github/ISSUE_TEMPLATE/rfc.yml]: do*

File: .github/ISSUE_TEMPLATE/rfc.yml [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+  validations:
+    required: true
+- type: textarea
+  attributes:
+    label: Alternatives Considered
+    description: A list of alternatives, that have been considered and offer equal or similar features to the proposed change.
+    placeholder: Maybe the cat could be replaced with an axolotl?
```
Comment (ghost) [inline .github/ISSUE_TEMPLATE/rfc.yml:58]: replace axolotl with sneed

**Review by ghost — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/rfc.yml (hunk 1 above)
Comment (d-513) [reply .github/ISSUE_TEMPLATE/rfc.yml]: no

File: .github/ISSUE_TEMPLATE/rfc.yml (hunk 2 above)
Comment (d-513) [reply .github/ISSUE_TEMPLATE/rfc.yml:58]: no

File: .github/ISSUE_TEMPLATE/rfc.yml [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+    label: Goal
+    description: Short description, 1-2 sentences.
+    placeholder: Remove the cat from the launcher.
+  validations:
+    required: true
+- type: textarea
+  attributes:
+    label: Motivation
+    description: |
+      Introduce the topic. If this is a not-well-known section of Arch Linux, a detailed explanation of the background is recommended.
```
Comment (d-513) [inline .github/ISSUE_TEMPLATE/rfc.yml]:
```suggestion
      Introduce the topic. If this is a not-well-known section of PolyMC, a detailed explanation of the background is recommended.
```

**Review by d-513 — CHANGES_REQUESTED** [review]

**Review by timoreo22 — APPROVED** [review]
Cool !

**Review by d-513 — APPROVED** [review]

**Comment by ghost** [comment]
replace axolotl with sneed

**Review by ghost — APPROVED** [review]

**Pull request merged**